pulpcore/pulpcore/app/models/repository.py

```diff
@@ -205,9 +205,11 @@ class RepositoryContent(Model):
     content = models.ForeignKey('Content', on_delete=models.CASCADE,
                                 related_name='version_memberships')
     repository = models.ForeignKey(Repository, on_delete=models.CASCADE)
-    version_added = models.ForeignKey('RepositoryVersion', related_name='added_memberships')
+    version_added = models.ForeignKey('RepositoryVersion', related_name='added_memberships',
+                                      on_delete=models.CASCADE)  # TODO: correct behavior?
```

Comment:
I think it needs to be `PROTECT` as the repo version delete code cleans up these relationships.

Reply:
Actually, thinking this through some more, I think `CASCADE` is correct in order to support the case where a repository is deleted.